packages/suite/src/support/messages.ts

```diff
@@ -1851,9 +1851,18 @@ export default defineMessages({
         defaultMessage: 'Display rotation',
         id: 'TR_DEVICE_SETTINGS_DISPLAY_ROTATION',
     },
-    TR_DEVICE_SETTINGS_HOMESCREEN_IMAGE_SETTINGS: {
+    TR_DEVICE_SETTINGS_HOMESCREEN_IMAGE_SETTINGS_1: {
```

Comment:
these number suffix is really cryptic for me, can we use something more explicit like `_T1`?

Reply:
c26a19c9524aada4045628d1305c9bb6eb82941e